hedera-node/src/main/java/com/hedera/services/txns/consensus/TopicUpdateTransitionLogic.java

```diff
@@ -233,7 +233,7 @@ private void applyNewAutoRenewPeriod(ConsensusUpdateTopicTransactionBody op, Mer
 	}
 
 	private boolean designatesAccountRemoval(AccountID id) {
-		return id.getShardNum() == 0 && id.getRealmNum() == 0 && id.getAccountNum() == 0;
+		return id.getShardNum() == 0 && id.getRealmNum() == 0 && id.getAccountNum() == 0 && id.getAlias().isEmpty();
```

Comment:
This is repeated. Perhaps we should move this function to a utils class (`MiscUtils`)